build-config/scripts/apply-patch-series

```
@@ -18,7 +19,7 @@ PATCH_ROOT=`dirname $SERIES`
 cd "$SRC" &&
 git init && 
 git add . > /dev/null &&
-git commit -m "Unmodified $(basename $SRC) src" > /dev/null &&
+git commit -m "Unmodified $(basename $SRC) src" --allow-empty > /dev/null &&
```

why is this needed?  it seems like it might hide real failures or mistakes.